docs/4.2/gcp_guide.md

```diff
@@ -210,4 +210,4 @@ proxy_service:
 
 **4. Add Users**
 
-Follow [adding users](enterprise/quickstart-enterprise.md#adding-users) or integrate with [G Suite](ssh_gsuite.md) to provide SSO access.
+Follow [adding users](enterprise/quickstart-enterprise.md#adding-users) or integrate with [G Suite](/enterprise/sso/ssh_gsuite.md) to provide SSO access.
```

Comment:
Should probably be `enterprise/sso/ssh_gsuite.md` (i.e. relative link rather than absolute with the `/`)